code/WorkInProgress/laundry.dm

```diff
@@ -22,6 +22,7 @@ TYPEINFO(/obj/submachine/laundry_machine)
 	var/cycle_current = 0
 	var/cycle_max = CYCLE_TIME
 	var/mob/occupant = null
+	var/mob/activator = null
```

Comment:
clear these on disposing please

Reply:
also maybe when the record gets processed. this would leave a hanging ref to the mob.